geopandas/io/file.py

```diff
@@ -39,6 +39,14 @@ def _is_url(url):
         return False
 
 
+def _is_vsi(url):
+    """Check if a filepath/url is using a fiona vsi scheme."""
+    try:
+        return fiona.vfs.valid_vsi(parse_url(url).scheme)
+    except:
```

Comment:
Can you make this a more specific exception? 
Is it the problem that older versions of fiona don't have this method, so it will give an AttributeError?

Reply:
To be honest, I was just mimicking the check right above this one. I can make this specifically catch an `AttributeError` for older versions of Fiona.